src/holdem/parse.rs

```diff
@@ -69,6 +69,8 @@ enum RangeIterValueSpecifier {
     Static(Value),
     /// Pair
     Pair,
+    /// Sequential. Value is the higher value. Second value is the gap.
+    Sequential(Value, u8),
```

Comment:
I  think `StaticWithRange` or `StaticRange` are better names. These aren't hands where the cards are sequential.